libs/platform/src/lib/components/menu/menu.component.ts

```diff
@@ -90,7 +90,7 @@ export class MenuComponent implements OnInit, AfterViewInit, AfterContentInit, O
      */
     @Output() close: EventEmitter<MenuCloseMethod> = new EventEmitter();
 
-    private _direction: 'ltr' | 'rtl' | void = 'ltr';
+    public direction: 'ltr' | 'rtl' | void = 'ltr';
```

Comment:
In what situation `void` is an expected type?

Reply:
This was left over from when a different service was used for directionality. I'll change it.